dexmaker/build.gradle

```diff
@@ -14,17 +25,27 @@ dependencies {
     compileOnly project(':dalvikdx')
 }
 
+task proguardFiles(type: ProGuardTask, dependsOn: jar) {
+    verbose
+    forceprocessing
+    configuration files('proguard-rules.pro')
+    libraryjars files('C:/Program Files (x86)/Android/android-sdk/platforms/android-28/android.jar',tasks.getByPath(':dalvikdx:jar').outputs)
```

Comment:
the hard-coded path looks bad

Reply:
Updated.
I see ANDROID_HOME not redefined in ci script after using a external SDK. However this should work if CI system image already contain Android SDK.